### ISSUE TITLE
options.js ES6 variable visibility bug

### ISSUE THREAD

**Issue by DaAitch** [webpro/reveal-md#193]
# Bug report
## Environment
**Reveal-md 2.0.2**

_reveal-md.json_
```json
{
	"separator": "^\n\n\n",
  	"verticalSeparator": "^\n\n",
	"theme": "simple",
	"css": "style.css"
}
```

**command:** `reveal-md --highlight-theme github README.md `

## Description
When I run the command, I expect my README.md to be provided as a presentation (as in earlier versions), but I get an runtime error in JS.

```
.../node-v8.9.4-linux-x64/lib/node_modules/reveal-md/lib/options.js:119
    options.cssPaths = getAssetPaths(options.css);
                       ^

ReferenceError: getAssetPaths is not defined
    at parseOptions (.../node-v8.9.4-linux-x64/lib/node_modules/reveal-md/lib/options.js:119:24)
    at Object.<anonymous> (.../node-v8.9.4-linux-x64/lib/node_modules/reveal-md/lib/options.js:64:10)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
    at Module.require (module.js:587:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (.../node-v8.9.4-linux-x64/lib/node_modules/reveal-md/lib/index.js:4:27)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
    at Module.require (module.js:587:17)
```

Maybe this also applies to `getAssetSourcePath`, `getAssetSourcePaths` and `getAssetPath`, as they are also declared the same way.

Bug was implemented with [582451b](https://github.com/webpro/reveal-md/commit/582451b68d7566d11e171a491263e1289deca880#diff-763aa8deb604d725a74f8304903c1422)

## Possible Solution
- make `getAssetPaths` an old style js function
- declare ES6 `const` variable before `parseOptions`

**Comment by webpro** [comment]
Thanks for the bug report, @DaAitch!

Should be fixed with [v2.0.3](https://github.com/webpro/reveal-md/releases/tag/2.0.3).

**Comment by DaAitch** [comment]
thank you, diff looks good. will check it later
